### PR TITLE
adjust beforeUnload behavior

### PR DESCRIPTION
**Pull request by taulinger**
Relevant active tools are now  `[Tools.DRAW, Tools.MEASURE, Tools.MEASURE]` only

### CODE REVIEW

File: src/plugins/BeforeUnloadPlugin.js [hunk 1]
```diff
@@ -36,4 +37,8 @@ export class BeforeUnloadPlugin extends BaPlugin {
 			observe(store, (state) => state.tools.current, onToolChanged, false);
 		}
 	}
+
+	_getTools() {
+		return [Tools.DRAW, Tools.MEASURE, Tools.MEASURE];
```
Comment (thiloSchlemmer) [inline src/plugins/BeforeUnloadPlugin.js:42]: could be reduced to `[Tools.DRAW, Tools.MEASURE]`

**Review by thiloSchlemmer — COMMENTED** [review]
is there any need to double 'Tools.MEASURE'?

File: src/plugins/BeforeUnloadPlugin.js (hunk 1 above)
Comment (taulinger) [reply src/plugins/BeforeUnloadPlugin.js:42]: Thx must be `ROUTING` of course

**Review by thiloSchlemmer — APPROVED** [review]
Thx

**Pull request merged**